### ISSUE TITLE
Common processing interface for identity & training checks

### ISSUE THREAD

**Issue by kepaik** [MIT-LCP/physionet-build#1686]
It's odd to have a different interface on these two pages:

https://physionet.org/console/credential_processing/
https://physionet.org/console/training/review/

Key features:

1. Training page: Search box that filters results
2. Identity page: Sort options for all field headings

Notes

- Don't care about pagination, especially if we have the search box. Could keep it or not.
- Styling is inconsistent, which doesn't make a difference, but why not use the same style sheet?

**Comment by tompollard** [comment]
I agree that it would be good to make these pages more consistent. The pages are using the same CSS style sheets, but I can see there are some differences in the design/layout.

**Comment by kepaik** [comment]
Any update with this? Would help processing classes...
